static/styles/drafts.css

```diff
@@ -95,6 +95,10 @@
             border: 0;
         }
 
+        .messagebox-content {
+            padding: 4px 115px 1px 10px;
```

Comment:
How is the 115px here calculated? Could it be better done as a `calc()` or using a variable shared with a related value in this file?

Reply:
i copied these values from how the padding was originally on formats where this bug didn't occur:
![tmp](https://user-images.githubusercontent.com/74348920/184531478-0536fa8f-4cda-498c-b701-9bebaa895a5a.png)